OBAApplicationDelegate.m

```diff
@@ -52,103 +52,48 @@ - (id)init {
     self = [super init];
 
     if (self) {
-
         self.active = NO;
 
-        _references = [[OBAReferencesV2 alloc] init];
-        _modelDao = [[OBAModelDAO alloc] init];
-        _locationManager = [[OBALocationManager alloc] initWithModelDao:_modelDao];        
-                
-        _modelService = [[OBAModelService alloc] init];
-        _modelService.references = _references;
-        _modelService.modelDao = _modelDao;
-        
-        OBAModelFactory * modelFactory = [[OBAModelFactory alloc] initWithReferences:_references];
-        _modelService.modelFactory = modelFactory;
-        
-        _modelService.locationManager = _locationManager;
-        
         _stopIconFactory = [[OBAStopIconFactory alloc] init];
-        
-        [self refreshSettings];
+
+
+        self.regionObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kOBAApplicationSettingsRegionRefreshNotification
+                                                                                object:nil
+                                                                                 queue:[NSOperationQueue mainQueue]
+                                                                            usingBlock:^(NSNotification *note) {
+                                                                                self.regionHelper = [[OBARegionHelper alloc] init];
```

Comment:
Correct.   

The block captures self, the app delegate, which is really a singleton that lives for the lifetime of the application.    If you feel strongly about this I'll weakify it, but it doesn't make a difference one way or another.